t/Mailbox/query/basic.t

```diff
@@ -0,0 +1,485 @@
+use strict;
+use warnings;
+use Test::Routine;
+use Test::Routine::Util;
+
+with 'JMAP::TestSuite::Tester';
+
+use JMAP::TestSuite::Util qw(batch_ok pristine_test);
+
+use Test::Deep ':v1';
+use Test::Deep::JType;
+use Test::More;
+use JSON qw(decode_json);
+use JSON::Typist;
+
+# XXX - Need test for cancalc
+
+pristine_test "Mailbox/query with no existing entities" => sub {
+  my ($self) = @_;
+
+  plan skip_all => "Cyrus requires at least one mailbox"
```

Comment:
Again:  I think it is likely universal that there will be _one_ mailbox, at least.